packages/components/switch/__tests__/switch.test.tsx

```diff
@@ -185,7 +184,7 @@ describe('Switch.vue', () => {
     const coreWrapper = wrapper.find('.el-switch__core')
     const switchWrapper: VueWrapper<SwitchInstance> =
       wrapper.findComponent(Switch)
-    const switchVm = switchWrapper.vm
+    const switchVm: any = switchWrapper.vm
```

Comment:
Is it possible to avoid use the any?

Reply:
I've tried, but no way not to use any.

Reply:
Maybe expose `checked` from the `el-switch`?

Reply:
It seems that removing `:any` does not cause a type error.